exercises/codec-server/solution/codec.py

```diff
@@ -0,0 +1,27 @@
+from typing import Iterable, List
+
+import cramjam
+from temporalio.api.common.v1 import Payload
+from temporalio.converter import PayloadCodec
+
+
+class EncryptionCodec(PayloadCodec):
```

Comment:
General point — it's less that we're scared of choosing an encryption algorithm (I actually make some recommendations from our Slack in https://github.com/temporalio/edu-appdatasec-go-content/blob/main/compression-and-encryption/data-encryption-methods.md) and more that proper key management is extremely out of scope for the course, and any encryption samples we could provide that do more than obfuscate bytes are much more problematically incomplete.

Reply:
I don't think you need key management for an encryption sample, just something to show where they would apply encrypt/decrypt operations. I'd be ok if we even stubbed out the actual encryption to a "encryption" class whose "encrypt" and "decrypt" implementations are just base64. I think the primary goal should be to show "if you want encryption, here's how you plug it into Temporal" which I fear compression samples don't make clear.

Reply:
I think this is important to address. It can be unclear to a user to have a security course with no clear sample to implement security, only compression.

Reply:
Yeah, good point. Done in https://github.com/temporalio/edu-appdatasec-python-code/pull/7.